install-dev/data/db_structure.sql

```diff
@@ -1,2630 +1,2821 @@
+  `show_prices` tinyint(1) unsigned NOT NULL DEFAULT '1', 
+  `date_add` datetime NOT NULL, 
+  `date_upd` datetime NOT NULL, 
   PRIMARY KEY (`id_group`)
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Localized group info */
 CREATE TABLE `PREFIX_group_lang` (
-  `id_group` int(10) unsigned NOT NULL,
-  `id_lang` int(10) unsigned NOT NULL,
-  `name` varchar(32) NOT NULL,
-  PRIMARY KEY (`id_group`,`id_lang`)
+  `id_group` int(10) unsigned NOT NULL, 
+  `id_lang` int(10) unsigned NOT NULL, 
+  `name` varchar(32) NOT NULL, 
+  PRIMARY KEY (`id_group`, `id_lang`)
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Category specific reduction */
 CREATE TABLE `PREFIX_group_reduction` (
-	`id_group_reduction` MEDIUMINT UNSIGNED NOT NULL AUTO_INCREMENT,
-	`id_group` INT(10) UNSIGNED NOT NULL,
-	`id_category` INT(10) UNSIGNED NOT NULL,
-	`reduction` DECIMAL(4, 3) NOT NULL,
-	PRIMARY KEY (`id_group_reduction`),
-	UNIQUE KEY(`id_group`, `id_category`)
+  `id_group_reduction` MEDIUMINT UNSIGNED NOT NULL AUTO_INCREMENT, 
+  `id_group` INT(10) UNSIGNED NOT NULL, 
+  `id_category` INT(10) UNSIGNED NOT NULL, 
+  `reduction` DECIMAL(4, 3) NOT NULL, 
+  PRIMARY KEY (`id_group_reduction`), 
+  UNIQUE KEY(`id_group`, `id_category`)
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Cache which store product price after reduction */
 CREATE TABLE `PREFIX_product_group_reduction_cache` (
-	`id_product` INT UNSIGNED NOT NULL,
-	`id_group` INT UNSIGNED NOT NULL,
-	`reduction` DECIMAL(4, 3) NOT NULL,
-	PRIMARY KEY (`id_product`, `id_group`)
+  `id_product` INT UNSIGNED NOT NULL, 
+  `id_group` INT UNSIGNED NOT NULL, 
+  `reduction` DECIMAL(4, 3) NOT NULL, 
+  PRIMARY KEY (`id_product`, `id_group`)
 ) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Specify a carrier for a given product */
 CREATE TABLE `PREFIX_product_carrier` (
-  `id_product` int(10) unsigned NOT NULL,
-  `id_carrier_reference` int(10) unsigned NOT NULL,
-  `id_shop` int(10) unsigned NOT NULL,
-  PRIMARY KEY (`id_product`, `id_carrier_reference`, `id_shop`)
-) ENGINE = ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
+  `id_product` int(10) unsigned NOT NULL, 
+  `id_carrier_reference` int(10) unsigned NOT NULL, 
+  `id_shop` int(10) unsigned NOT NULL, 
+  PRIMARY KEY (
+    `id_product`, `id_carrier_reference`, 
+    `id_shop`
+  )
+) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
 
 /* Stats from guest user */
 CREATE TABLE `PREFIX_guest` (
-  `id_guest` int(10) unsigned NOT NULL auto_increment,
-  `id_operating_system` int(10) unsigned DEFAULT NULL,
-  `id_web_browser` int(10) unsigned DEFAULT NULL,
-  `id_customer` int(10) unsigned DEFAULT NULL,
-  `javascript` tinyint(1) DEFAULT '0',
-  `screen_resolution_x` smallint(5) unsigned DEFAULT NULL,
-  `screen_resolution_y` smallint(5) unsigned DEFAULT NULL,
-  `screen_color` tinyint(3) unsigned DEFAULT NULL,
-  `sun_java` tinyint(1) DEFAULT NULL,
-  `adobe_flash` tinyint(1) DEFAULT NULL,
-  `adobe_dire
```

Comment:
This column does not match the definition in `Product.php`

Reply:
Fixed

Reply:
I did not change anything related to this in my PR :-)

Reply:
If there is an issue, can this be fixed in an other PR?

Reply:
You actually did :)

https://github.com/PrestaShop/PrestaShop/pull/10446/files#diff-0a7a17e9ff155e6890d7a5b01eea713cR391

Reply:
I just updated meta_title in this commit: https://github.com/PrestaShop/PrestaShop/pull/10446/commits/b1486074c8a22985e2f0135235816bb3dada3cd8 :) But I did not change anything related to `adobe_dire`